docs/src/main/paradox/scala/http/low-level-server-side-api.md

```diff
@@ -118,15 +128,15 @@ connection. An often times more convenient alternative is to explicitly add a `C
 `HttpResponse`. This response will then be the last one on the connection and the server will actively close the
 connection when it has been sent out.
 
-Connection will also be closed if request entity has been cancelled (e.g. by attaching it to `Sink.cancelled`)
+Connection will also be closed if request entity has been cancelled (e.g. by attaching it to @scala[`Sink.cancelled`]@java[`Sink.cancelled()`])
```

Comment:
I believe that the Scala version allows to use parentheses which is probably also better style given that it has side effects.